### ISSUE TITLE
why the result is wrong

### ISSUE THREAD

**Issue by hy05190134** [openresty/lua-resty-string#47]
when I use 128_ecb,  first encrypt the content, then base64, why the result is wrong, is it exist a bug that it default use md5 to encode the key then use to encrypt? I just want to use the origin key

**Comment by hy05190134** [comment]
@agentzh

**Comment by haorenfsa** [comment]
yes, it does not use your input key by default. I've tried to give an solution, see my pull request #46
